integration-tests/src/test/java/org/openmrs/maven/plugins/AbstractSdkIT.java

```diff
@@ -171,6 +171,22 @@ public String setupTestServer(String distro) throws Exception{
         return serverId;
     }
 
+    public void setupContentPackage(String contentPackageDir) throws Exception {
+        Path sourcePath = testResourceDir.resolve(TEST_DIRECTORY).resolve("contentpackages").resolve(contentPackageDir);
+        Path targetPath = testDirectoryPath.resolve("contentpackages").resolve(contentPackageDir);
+        FileUtils.copyDirectory(sourcePath.toFile(), targetPath.toFile());
+        Verifier setupContentPackage = new Verifier(targetPath.toFile().getAbsolutePath());
+        try {
+            addAnswer("y");
+            addTaskParam(setupContentPackage, BATCH_ANSWERS, getAnswers());
+            setupContentPackage.executeGoal(resolveSdkArtifact() + ":build");
+        }
+        finally {
+            cleanAnswers();
+            verifier.resetStreams();
+        }
+    }
+
```

Comment:
This allows us to create any content packages we wish for testing, without relying on published versions in the cloud